ompi/mpi/java/c/mpi_Info.c

```diff
@@ -45,17 +45,20 @@ JNIEXPORT jlong JNICALL Java_mpi_Info_getNull(JNIEnv *env, jclass clazz)
     return (jlong)MPI_INFO_NULL;
 }
 
+// At least some versions of jni.h have a global named "jvalue", and
+// we get a compiler warning if we have a parameter or variable of the
+// same name.  So use "ljvalue" instead.
 JNIEXPORT void JNICALL Java_mpi_Info_set(
-        JNIEnv *env, jobject jthis, jlong handle, jstring jkey, jstring jvalue)
+        JNIEnv *env, jobject jthis, jlong handle, jstring jkey, jstring ljvalue)
 {
-    const char *key   = (*env)->GetStringUTFChars(env, jkey,   NULL),
-               *value = (*env)->GetStringUTFChars(env, jvalue, NULL);
+    const char *key   = (*env)->GetStringUTFChars(env, jkey,    NULL),
```

Comment:
why the extra spaces before NULL?

Reply:
I just maintained the same spacing that was there from the original author: that the NULL's on the 2 successive lines were aligned.

I don't know if this style is uniform throughout the Java bindings, but it does occur in at least a few places that I spot checked.  Want me to remove them here?  I don't have strong feelings, either way.

Reply:
nah, don't worry about it.  Although in the diff, it looks like you added a space.